common/utilities/integration_adaptors_logger.py

```diff
@@ -102,6 +102,15 @@ def _format_values_in_map(self, dict_values: dict) -> dict:
         Replaces the values in the map with key=value so that the key in a string can be replaced with the correct
         log format, also surrounds the value with quotes if it contains spaces and removes spaces from the key
         """
+
+        for key in dict_values.keys():
+            if str(key).lower() == "exception":
```

Comment:
I'd maybe add to one of the higher level docs (Maybe the `__init__`) that exceptions should be keyed as `{exception}` so that someone doesn't have to dig down to this line to find out that exceptions won't print correctly if they aren't keyed appropriately